tests/Behat/OAuthContext/OAuthContext.php

```
@@ -139,30 +150,45 @@
+        $this->setFormUrlEncodedHeaders();
+
+        if (!isset($this->obtainAccessTokenInput)) {
+            throw new RuntimeException(
+                'obtainAccessTokenInput is not set. '
+                . 'Call the corresponding Given step before requesting a token.'
+            );
+        }
+
+        $requestData = array_filter(
+            get_object_vars($this->obtainAccessTokenInput),
+            static fn ($v) => $v !== null
```

PHPMD (2.15.0)

Avoid variables with short names like $v. Configured minimum length is 3. (Naming Rules)

(ShortVariable)